src/NuGet.Jobs.Auxiliary2AzureSearch/README.md

```diff
@@ -0,0 +1,110 @@
+## Overview
+
+**Subsystem: Search 🔎**
+
+This job updates the [search auxiliary files](../../docs/Search-auxiliary-files.md) used by the [search service](../NuGet.Services.SearchService). This also updates the downloads data in the [Azure Search "search" index](../../docs/Azure-Search-indexes.md).
```

Comment:
I would say it updates the downloads **and** owners data

Reply:
Oops good catch

Reply:
Fixed